src/libraries/System.Text.Json/tests/JsonNode/JsonValueTests.cs

```diff
@@ -0,0 +1,291 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Text.Json.Serialization;
+using Xunit;
+
+namespace System.Text.Json.Node.Tests
+{
+    public static class JsonValueTests
+    {
+        [Fact]
+        public static void CreateFromNull()
+        {
+            Assert.Null(JsonValue.Create((object)null));
+        }
+
+        [Fact]
+        public static void CreateFromNode_Fail()
+        {
+            Assert.Throws<ArgumentException>(() => JsonValue.Create(new JsonArray()));
+            Assert.Throws<ArgumentException>(() => JsonValue.Create(new JsonObject()));
+            Assert.Throws<ArgumentException>(() => JsonValue.Create(JsonValue.Create(42)));
+        }
+
+        [Fact]
+        public static void GetValue_Throws()
+        {
+            Assert.Throws<InvalidOperationException>(() => JsonNode.Parse("{}").GetValue<object>());
+            Assert.Throws<InvalidOperationException>(() => JsonNode.Parse("[]").GetValue<object>());
+        }
```

Comment:
This doesn't seem to apply directly to the `JsonValue` type itself. Should this test be in `JsonNodeTests.cs`?

Reply:
yes